lib/timex.ex

```diff
@@ -633,7 +633,6 @@ defmodule Timex do
       {year, month, day} = Helpers.iso_day_to_date_tuple(year, iso_day)
       %Date{year: year, month: month, day: day}
   end
-  def from_iso_triplet(_, _, _), do: {:error, {:from_iso_triplet, :invalid_triplet}}
```

Comment:
It should be `def from_iso_triplet({_,_,_}), do: {:error, ...}`, that's why it appears that there is no useful clause. Could you update this PR with that change? I'll merge it once it is. Thanks!